Documentation/ceph-cluster-crd.md

```diff
@@ -208,6 +207,29 @@ Some modules will have special configuration to ensure the module is fully funct
 
 * `pg_autoscaler`: Rook will configure all new pools with PG autoscaling by setting: `osd_pool_default_pg_autoscale_mode = on`
 
+### Network Configuration Settings
+
+Configure the network that will be enabled for the cluster and services. This is optional and if not defined then the default SDN will be used.
+
+* `provider`: Specifies the network provider that will be used to connect the network interface. You can choose between `host`, and `multus`.
+* `selectors`: List the network selector(s) that will be used associated by a key.
+
+The selector keys are hard-coded to `public` and `cluster`.
+Based on the configuration, the operator will do the following:
+
+  1. if only the `public` selector is specified both `public_network` and `cluster_network` Ceph settings will listen on that interface
+  2. if both `public` and `cluster` selectors are specified the first one will point to `public_network` flag and the second one to `cluster_network`
+
+In order to work, each selector value must match a `NetworkAttachmentDefinition` object in Multus.
+The following are just arbitrary examples and not hardcoded names.
```

Comment:
```suggestion
The following are just arbitrary examples and not required names.
```

Reply:
wait, aren't `public` and `cluster` still required? Or you mean the names that come in the mapping aren't required.

Reply:
The value, clarified.